projects/igniteui-angular/src/lib/grids/grid/grid.component.spec.ts

```diff
@@ -1692,6 +1692,28 @@ describe('IgxGrid Component Tests #grid', () => {
         });
     });
 
+    describe('IgxGrid - footer with no paging', () => {
```

Comment:
You don't necessarily need a separate describe and test component just for this test. You can re-use IgxGridWithCustomFooterComponent and use a prop model binding to change the [paging] input to enable/disable the paging in the test.

Reply:
Done:)